src/main/java/io/gatling/demostore/WebConfig.java

```diff
@@ -14,11 +14,6 @@ public void addResourceHandlers(ResourceHandlerRegistry registry) {
                 .addResourceHandler("/resources/**")
                 .addResourceLocations("classpath:/media/")
                 .addResourceLocations("classpath:/downloads/");
-
-        registry
-                .addResourceHandler("/swagger-ui/**")
-                .addResourceLocations("classpath:/META-INF/resources/webjars/springfox-swagger-ui/")
-                .resourceChain(false);
```

Comment:
Doesn't seem needed, I assume `springdoc-openapi-ui` handles this automagically 🤷